### PR TITLE
l1 hometask

### PR DESCRIPTION
**Pull request by var-bin**
l1 hometask complete. I need code review and feedback.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,5 +5,11 @@
     "var-bin <rashan_soft@inbox.ru>"
   ],
   "description": "",
-  "main": ""
+  "main": "",
+  "dependencies": {
+    "underscore": "1.8.3",
+    "backbone": "1.x",
+    "react": "> 0.1.0",
+    "angular": "^1.5.5"
```
Comment (puzankov) [inline bower.json]: angular любой версии в задании было

File: client_src/js/script.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        }
+
+        // if options get only 1 argument
+        if (optionsCount && optionsCount == 1) {
+          if (data[i].hasOwnProperty(options[0])) {
+            data[i][options[0]] = hdb.Utils.escapeExpression(data[i][options[0]]);
+          }
+        }
+      });
+
```
Comment (puzankov) [inline client_src/js/script.js]: overengineering

Comment (puzankov) [inline client_src/js/script.js]: need skype call

File: client_src/libs/angular/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# packaged angular
```
Comment (puzankov) [inline client_src/libs/angular/README.md]: all libs need to be in .gitignore

File: less_components/vars.less [hunk 4]
```diff
@@ -0,0 +1 @@
+// less_components/vars.less
```
Comment (puzankov) [inline less_components/vars.less]: empty file

**Comment by var-bin** [comment]
I will fix you comments. After I will create new PR

**Pull request closed without merging**